app/jobs/scheduled/automatic_translation_backfill.rb

```diff
@@ -87,26 +87,35 @@ def translate_records(type, record_ids, target_locale)
 
     def process_batch
       records_to_translate = SiteSetting.automatic_translation_backfill_rate
-      backfill_locales.each_with_index do |target_locale, i|
-        topic_ids =
-          fetch_untranslated_model_ids(Topic, "title", records_to_translate, target_locale)
-        post_ids = fetch_untranslated_model_ids(Post, "raw", records_to_translate, target_locale)
-
-        next if topic_ids.empty? && post_ids.empty?
-
-        DiscourseTranslator::VerboseLogger.log(
-          "Translating #{topic_ids.size} topics and #{post_ids.size} posts to #{target_locale}",
-        )
+      backfill_locales.each do |target_locale|
+        [
+          [Topic, "title"],
+          [Post, "raw"],
+          [Category, "name"],
```

Comment:
There is `Category#description` too. Is that something we want to consider in this PR too?